src/components/shared/card.tsx

```diff
@@ -22,23 +22,37 @@ interface Props {
     image: Option<Image>;
 }
 
-const listStyles = css`
-    background: white;
-    margin-right: ${remSpace[3]};
-    flex: 0 0 15rem;
-    height: 100%;
-    display: flex;
-    flex-direction: column;
-    justify-content: space-between;
-
-    img {
-        width: 100%;
-        height: 100%;
-        position: absolute;
-        top: 0;
-        left: 0;
+const labPillarColors = (itemType: RelatedItemType, format: Format): SerializedStyles => {
+    if (itemType === RelatedItemType.ADVERTISEMENT_FEATURE){
+        return css`1px solid ${palette.labs[300]}`
+    } else {
+        return css`1px solid ${getPillarStyles(format.pillar).kicker}`
     }
-`;
+}
+
+const listStyles = (itemType: RelatedItemType, format: Format): SerializedStyles => {
+    const { inverted } = getPillarStyles(format.pillar);
+        return css`
+                background: white;
+                margin-right: ${remSpace[3]};
+                flex: 0 0 15rem;
+                height: 100%;
+                display: flex;
+                flex-direction: column;
+                justify-content: space-between;
+                border-top : ${labPillarColors(itemType, format)};
+                ${darkModeCss`
```

Comment:
Can this be removed?

Reply:
I assume you mean the dark mode? as the border top is essential?

Reply:
yeah I think the dark mode styles are being overridden at the moment